packages/eslint-plugin/lib/configs/a11y.js

```diff
@@ -7,5 +7,10 @@ module.exports = defineConfig({
   extends: ["plugin:react-native-a11y/all"],
   rules: {
     "react-native-a11y/has-accessibility-hint": "off",
+    "@bam.tech/image-requires-accessible-prop": "error",
```

Comment:
I don't remember, why are the rules prefixed with @bam.tech ?

Reply:
The name of the package is `@bam.tech/eslint-plugin` as written in package.json.
To add a custom rule from a plugin, you would have to add the line: `plugins: ["@bam.tech/eslint-plugin"]`, then mention the rules you want to use from the latter in the `rules` section, with the norm: `<name-of-the-plugin>/<name-of-the-rule>.

As those configs are in the plugin, you don't have to add the `plugins` array. Furthermore, you don't have to add the `/eslint-plugin` as it is mandatory to name your plugin with this suffix, but then you can omit it in the eslint config.